mysos/scheduler/launcher.py

```diff
@@ -266,6 +272,13 @@ def _new_task(self, offer, task_cpus, task_mem, task_disk, task_port):
     task.name = task_id
 
     task.executor.executor_id.value = task_id  # Use task_id as executor_id.
+    task.executor.name = EXECUTOR_NAME
```

Comment:
Hmm, I think I prefer a static name than a variable one and it's (slightly) simpler to tell the monitoring tooling (folks who write the monitoring tooling) to watch for a full string than a prefix. `task_id` is already exposed through the API s/endpoints and `executor.name` doesn't have to be redundant.